docs/src/architecture.md

```diff
@@ -102,14 +110,13 @@ CloudNativePG clusters suffer a failure.
 This scenario is typical of self-managed on-premise Kubernetes clusters, where
 only one data center is available.
 
-Single availability zone Kubernetes is unfortunately the only viable option
-where just **two (2) data centers** are available within reach of a low latency
-connection (normally in the same metropolitan area): having only two zones
-precludes users from creating a multi-availability zone Kubernetes cluster
-(as the minimum number of
-3 zones is not reached) and forces them to create two different Kubernetes
-clusters in an active/passive configuration, where the second cluster is used
-primarily for Disaster Recovery.
+A single availability zone Kubernetes cluster is the only viable option when
+only **two data centers** are available within reach of a low-latency
+connection (typically in the same metropolitan area). Having only two zones
+prevents the creation of a multi-availability zone Kubernetes cluster, which
+requires a minimum of three zones. As a result, users must create two separate
+Kubernetes clusters in an active/passive configuration, with the second cluster
+primarily used for Disaster Recovery (see the [replica cluster feature](replica_cluster.md)).
```

Comment:
This paragraph confuses me.
We start by saying
(1) that with only two data centers available, a sing Kubernetes cluster is the only way
And then in the end we say that the user must create two Kubernetes clusters in active/passive.
So, it seems, we go back on (1)

I think there are three different points:
1 - with two data centers in low-latency config, only a single k8s cluster is viable
2 - two data centers in separate AZs are not enough to create an HA K8s cluster
3 - if you do have two data centers, then you have to create two K8s clusters